TombLib/TombLib/LevelData/Compilers/TombEngine/Textures.cs

```diff
@@ -63,14 +67,14 @@ private void BuildSprites()
             var spriteSequences = _level.Settings.WadGetAllSpriteSequences();
 
             // Add all sprites to the texture packer
-            var spriteAllocator = new Util.SpriteAllocator(_level.Settings.GameVersion == TRVersion.Game.TombEngine ? 2048 : 256);
+            var spriteAllocator = new TombEngineSpriteAllocator(4096);
```

Comment:
Why there is a magic number of 4096? You should be able to get texture page size from `_limits` like this: `_limits[Limit.TexPageSize]`.

Reply:
Done!